build/Common.props

```diff
@@ -0,0 +1,16 @@
+<Project>
+  <PropertyGroup>
+    <LangVersion>8.0</LangVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Package versions used in this repository">
+    <MicrosoftNETFrameworkReferenceAssembliesPkgVer>[1.0.0]</MicrosoftNETFrameworkReferenceAssembliesPkgVer>
```

Comment:
I think eventually we want all to extract package version from individual projects to this central file.
We probably should have another place (e.g. `Common.test.props`) for test dependency.